README.md

```diff
@@ -0,0 +1,43 @@
+# CAOTIC
+High Contrast and Adaptive Optics laboratory testbeds community website
+
+Welcome to the community website for astronomical testbeds! We are currently working on getting our 
+website migrated to highconaotools.github.io, the original website can be found here:  
+https://sites.google.com/view/highcontrastlabs/
+
+## Local setup - MacOS
+
+### Initial setup
+
+If you use conda, make sure you have deactivated all your environments (including `base`) by doing `$ conda deactivate` often enough.
+
+*On a Mac*:
+1. Install homebrew following the instructions on this page:  
+https://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/
+
+2. Install the package manager rbenv (more on rbenv [here](https://github.com/rbenv/rbenv) ) 
+`$ brew install rbenv`
+
+3. Set up rbenv in your shell  
+`$ rbenv init`
+
```

Comment:
I get the following prompt when I run this:

```
# Load rbenv automatically by appending
# the following to ~/.bash_profile:

eval "$(rbenv init -)"
```

It might make more sense to make what I wrote above step 3?

Reply:
It's been too long again, what was the issue with this step again?

Reply:
If you just type `$ rbenv init` in your terminal, you get the prompt that I gave above, so I am suggesting that we just add a step to add `eval "$(rbenv init -)" ` to the bash_profile. I am not sure I see the point of just running `$ rbenv init` if it only works the one time (which is what I deduce but have no idea if that is actually the case)

Reply:
ok cool, I have added this to the readme now